app/src/main/java/org/fossasia/openevent/app/common/data/models/Event.java

```diff
@@ -32,10 +44,18 @@
 @AllArgsConstructor
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
 @Table(database = OrgaDatabase.class, allFields = true)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
```

Comment:
Why is this needed?

Reply:
This was added to reject the fields with non default values such that the request json doesn't contain unwanted stuff but it don't affect the functionality of the app i.e. event can be created without it. So i'll remove it.